src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -351,32 +355,6 @@ class IpexCommunicationService extends AgentService {
     return op;
   }
 
-  async markAcdc(
-    credentialId: string,
-    status: CredentialStatus.CONFIRMED | CredentialStatus.REVOKED
-  ) {
-    const metadata = await this.credentialStorage.getCredentialMetadata(
-      credentialId
-    );
-    if (!metadata) {
-      throw new Error(
-        IpexCommunicationService.CREDENTIAL_MISSING_METADATA_ERROR_MSG
```

Comment:
This error can be removed now from this file

Reply:
@Sotatek-TungNguyen2a Please revert the changes to connectionService. I was not asking to remove this check in that file.

I'm saying in this file - `ipexCommunicationService.ts` that the `IpexCommunicationService.CREDENTIAL_MISSING_METADATA_ERROR_MSG` error message is no longer used so can be removed and cleaned up.

Reply:
Sorry for my misunderstanding, i have reverted the changes to connectionService